docs/source/conf.py

```diff
@@ -78,12 +78,11 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
-
 html_theme = 'sphinx_rtd_theme'
 
 html_theme_options = {
-    'logo_only': True,
-    'navigation_depth': 3
+    'logo_only': True,  #
+    'navigation_depth': 3  #
```

Comment:
What is the purpose of these comments?

Reply:
to keep the lines separately - YAPF merges them